examples/java/core/enumerations/src/main/bond/enumerations.bond

```diff
@@ -0,0 +1,21 @@
+namespace Examples;
+
+enum Color
+{
+    Orange = 1,
+    Yellow = 2,
+}
+
+enum Fruit
+{
+    Apple = 1,
+    Orange = 2,
+}
+
+enum Limits
+{
+    Int32Min = -2147483648,
+    Int32Max = 2147483647,
+    UInt32Min = 0,
+    UInt32Max = 0xFFFFFFFF,
```

Comment:
It's a direct port of the C# example, using the same Bond IDL. What do you propose?